### ISSUE TITLE
suggest a walkthrough jupyter notebook

### ISSUE THREAD

**Issue by amiltonwong** [thu-ml/zhusuan#40]
Hi, authors,

It would be very helpful if there's walkthrough jupyter notebook like this [one](http://nbviewer.jupyter.org/github/blei-lab/edward/blob/master/notebooks/iclr2017.ipynb)
.
Thanks!

**Comment by thjashin** [comment]
Yes. That's in plan. Before that you could refer to our step-by-step tutorials http://zhusuan.readthedocs.io/en/latest/vae.html

**Comment by rwayan** [comment]
Any Lifeful example for beginner?

**Comment by thjashin** [comment]
@rwayan What kind of examples do you expect most? We may add some priority to it.

**Comment by rwayan** [comment]
我想应该可以说中文吧，比如说从生活中的一些例子，比如说什么煤矿灾难次数啦，通过分析得到一个什么结论啦，这样可以让更多非学术的朋友接触到这方便的知识。这样通过一个notebook直接可以传播，很不错啊。

**Comment by thjashin** [comment]
Be professional, potential contributors may not say Chinese :)

**Comment by fanbingjian** [comment]
When I see this very useful tool,I just want to say"Oh,my god"
